dbux-data/src/impl/indexes/TracesByStaticTraceIndex.js

```diff
@@ -9,8 +9,8 @@ export default class TracesByStaticTraceIndex extends CollectionIndex {
     super('traces', 'byStaticTrace');
   }
 
-  beforeAdd(ofKey, trace) {
-    trace.staticTraceIndex = ofKey.length;
+  beforeAdd(container, trace) {
```

Comment:
I think `ofKey` was a very clear name, while `container` is more ambiguous, generic.
But it's fine, we can leave as-is.
Just wanted to explain (parts of) how I think about names :)

Reply:
Cuz `ofKey` was a little confusing to me and we used `container` in the discussion last time so I changed it :P